src/main/java/io/seqera/tower/cli/commands/global/PaginationOptions.java

```diff
@@ -37,20 +37,13 @@ public static class Pageable {
     public static class Sizeable {
         @CommandLine.Option(names = {"--max"}, description = "Maximum number of records to display [default: " + MAX + "].")
         public Integer max;
-
-        @CommandLine.Option(names = {"--no-max"}, description = "Show all records.")
-        public Boolean noMax;
     }
 
     public static Integer getMax(PaginationOptions paginationOptions) throws TowerException {
         Integer max = PaginationOptions.MAX;
 
         if (paginationOptions.sizeable != null) {
-            if (paginationOptions.sizeable.noMax != null && paginationOptions.sizeable.max != null) {
-                throw new TowerException("Please use either --no-max or --max as pagination size parameter");
-            }
-
-            max = paginationOptions.sizeable.noMax != null ? null : paginationOptions.sizeable.max != null ? paginationOptions.sizeable.max : max;
+            max = paginationOptions.sizeable.max != null ? paginationOptions.sizeable.max : max;
```

Comment:
What's preventing the user to provide a stupidly large number for max?

Reply:
Nothing at the CLI. The backend has a maximum maximum, so even if they request for a big number they will get only the backend maximum. This backend maximum is configurable at backend side so we do not known it a priori at CLI side and cannot do a warning or error

Reply:
ok

Reply:
Somehow related to this I've opened this issue: https://github.com/seqeralabs/tower-cli/issues/223